daemon/cmd/hubble.go

```diff
@@ -131,17 +124,22 @@ func (d *Daemon) launchHubble() {
 	}()
 
 	// configure another hubble instance that serve fewer gRPC services
-	if len(addresses) > 0 {
+	address := option.Config.HubbleListenAddress
+	if address != "" {
+		// TODO: remove warning once mutual TLS has been implemented
+		if !strings.HasPrefix(address, "unix://") {
+			logger.WithField("address", address).Warn("Hubble server will be exposing its API insecurely on this address")
+		}
 		srv, err := server.NewServer(logger,
-			serveroption.WithListeners(addresses),
+			serveroption.WithListeners([]string{address}),
```

Comment:
Thinking about it, I don't see a use-case for the ability to listen on an additional Unix domain sockets. Thus, I propose the following: remove `WithListeners()` in `pkg/hubble/server/serveroption/option.go` and change this call for `WithTCPListener()` (it will fail if it's not a TCP socket).

I think we should also get rid of the `Listeners` map in `Options` and simply have `TCPListener` and `UnixSocketListener` members  as keeping a map for at most 2 values is overkill.

Reply:
will address in a follow-up pr.

Reply:
or in this pr: https://github.com/cilium/cilium/pull/11330/commits/9deccf928900ba79612918570665687ab4c950a8